tests/desktop/test_details_page.py

```diff
@@ -45,6 +45,7 @@ def test_that_about_this_addon_is_displayed(self, base_url, selenium):
 
     @pytest.mark.action_chains
     @pytest.mark.nondestructive
+    @pytest.mark.xfail(reason='https://github.com/mozilla/addons-server/issues/5135')
```

Comment:
By xfailing this we're losing quite a bit of coverage that should still pass. Ideally we'd only check one thing per test, but this test checks a number of things. Perhaps comment out (or even remove) the failing line. Especially as the test method name doesn't specify the view source feature.